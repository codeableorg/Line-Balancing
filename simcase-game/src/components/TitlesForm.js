/** @jsx jsx */
import { jsx } from "@emotion/core";

const container = {
  background: "rgb(213,245,245)",
  borderRadius: "4px",
  display: "table",
  tableLayout: "fixed",
  width: "90%",
  height: "4vh",
  boxSizing: "border-box",
  marginLeft: "auto",
  marginRight: "auto",
  marginTop: "16px", //Needs a little fix
  marginBottom: "16px" // Must be automatic
};

const item = {
  display: "table-cell",
  verticalAlign: "middle",
  textAlign: "center",
  fontFamily: "Arial",
  color: "blue",
  fontSize: "1.0em"
};

function TitlesForm() {
  return (
    <div css={container}>
      <div css={item}>Tasks</div>
      <div css={item}>Station 1</div>
      <div css={item}>Station 2</div>
      <div css={item}>Station 3</div>
    </div>
  );
}

export default TitlesForm;
