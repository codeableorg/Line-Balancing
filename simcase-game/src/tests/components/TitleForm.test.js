import React from "react";
import { render } from "@testing-library/react";

import TitlesForm from "../../components/TitlesForm";
test("TitlesForm based on game stage", () => {
  const { asFragment } = render(<TitlesForm />);
  expect(asFragment()).toMatchSnapshot();
});
