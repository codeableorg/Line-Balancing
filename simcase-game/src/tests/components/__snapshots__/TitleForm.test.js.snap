// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitlesForm based on game stage 1`] = `
<DocumentFragment>
  <div
    class="css-19ixip1-TitlesForm"
  >
    <div
      class="css-ec7gzg-TitlesForm"
    >
      Tasks
    </div>
    <div
      class="css-ec7gzg-TitlesForm"
    >
      Station 1
    </div>
    <div
      class="css-ec7gzg-TitlesForm"
    >
      Station 2
    </div>
    <div
      class="css-ec7gzg-TitlesForm"
    >
      Station 3
    </div>
  </div>
</DocumentFragment>
`;
